lib/generate.js

```diff
@@ -83,6 +83,7 @@ function generateScripts () {
     let publicPath = `${CONFIG.root}/${CONFIG.publicDir}`;
 
     let compiler = webpack({
+        mode: 'production',
```

Comment:
webpack 4 defaults to `production` and strongly recommends explicitly setting this configuration